README.Rmd

```diff
@@ -40,100 +40,31 @@ admiralci_homepage <- "https://pharmaverse.github.io/admiralci"
 <!-- The purpose of this is to enable dynamic links using dyn_link function above to access devel/main admiral homepage respectively -->
 <!-- To test this in your feature branch use code: rmarkdown::render("README.Rmd", output_format ="md_document") -->
 
-# admiral package extension Template <img src="man/figures/logo.png" align="right" width="200" style="margin-left:50px;"/>
+# pharmaverseadam <img src="man/figures/logo.png" align="right" width="200" style="margin-left:50px;"/>
 
 <!-- badges: start -->
 [<img src="http://pharmaverse.org/shields/admiral.svg">](https://pharmaverse.org)
 [![CRAN status](https://www.r-pkg.org/badges/version/admiral)](https://CRAN.R-project.org/package=admiral)
 [![Test Coverage](https://raw.githubusercontent.com/pharmaverse/admiraltemplate/badges/main/test-coverage.svg)](https://github.com/pharmaverse/admiraltemplate/actions/workflows/code-coverage.yml)
 <!-- badges: end -->
 
-## Table of Contents
+Test data (ADaM) for the pharmaverse family of packages
 
-- `r link("Extension Expectations", "#extension-expectations")`
-- `r link("What is included in the template", "#what-is-included-in-the-template")`
-- `r link("Package Extension Guidance", "#package-extension-guidance")`
-- `r link("Quick Start Guide for Template", "#quick-start-guide-for-template")`
+# Purpose
 
-## Extension Expectations 
+To provide a one-stop-shop for ADaM test data in the pharmaverse family of packages. This includes test ADaM data created from {admiral} templates. 
```

Comment:
Does "{admiral} template" refer to the templates in admiral core only or does it include the templates in the extension packages {admiralonco}, {admiralophta}, {admiralvaccines}, ...?

If it includes extension packages, we may need to add something regarding the requirements for the CI/CD workflow to process them.

Reply:
I think it includes admiral extension packages as well. @manciniedoardo will we have CI/CD ready for extension packages in this release? if not, @bundfussr can we add more details when the workflow is ready?

Reply:
initial setup is only for admiral but then extension packages can be included too in the near future

Reply:
> if not, @bundfussr can we add more details when the workflow is ready?

Yes, of course.

Reply:
thank you both! a separate issue created: https://github.com/pharmaverse/pharmaverseadam/issues/8